code/code/disc/disc_shaman.cc

```diff
@@ -1050,8 +1046,7 @@ int cheval(TBeing *caster, TBeing *victim, int level, short bKnown)
     aff.type = SPELL_CHEVAL;
     aff.level = level;
     aff.duration = caster->durationModify(SPELL_CHEVAL, (aff.level / 3) * Pulse::UPDATES_PER_MUDHOUR);
-    aff.modifier = -50;
-    aff.location = APPLY_ARMOR;
+    aff.location = APPLY_NONE;
```

Comment:
So what'll it do?

Reply:
The main point of cheval is reducing command lockout: https://github.com/sneezymud/sneezymud/blob/master/code/code/misc/utility.cc#L1569